bitmap-mosnter/test/bitmaptest.js

```diff
@@ -0,0 +1,115 @@
+    it('should return BM from file header', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(testobject.bheader.toString('utf8')).to.equal('BM');
+        done();
+      });
+    });
+
+    it ('should return 11078 as the size', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(testobject.size).to.equal(11078);
+        done();
+      });
+    });
+
+    it ('should return 256 color', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(testobject.colors).to.equal(256);
+        done();
+      });
+    });
+
+    it('should return 256 important colors', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(testobject.importantcolors).to.equal(256);
+        done();
+      });
+    });
+
+    it('should return a width of 100', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(testobject.width).to.equal(100);
+        done();
+      });
+    });
+
+    it('should return a height of 100',  function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(testobject.height).to.equal(100);
+        done();
+      });
+    });
+
+    it('should return a bits per pixel value of 8', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(testobject.bitsperpixel).to.equal(8);
+        done();
+      });
+    });
+  });
+});
+describe('testing the randomize transform', function() {
+  describe('Testing the output of the randomize function', function() {
+    it('output a string', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(transform.randomcolor(testobject)).to.be.a('string');
+        done();
+      });
+    });
+  });
+});
+
+describe('testing the randomize transform', function() {
+  describe('Testing the output of the randomize function', function() {
+    it('should have 6 characters', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        var testobject = new bitobject.Buffobject(data);
+        expect(transform.randomcolor(testobject)).to.have.length.of(6);
+        done();
+      });
+    });
+  });
+});
```

Comment:
Good job with these tests!  There's no reason to keep these in separate describe blocks though. c: